scalableminds/webknossos
# Restore Links from Projects to Task View

### Mailable description of changes:
- [Admin.Projects] Restored Link to see all Tasks of a Project
- [Admin.Tasks] Show a loading spinner when querying for tasks

### Steps to test:
- Navigate to Admin --> Projects
- Click on the "Tasks" Link in the "Action" column and be redirected to task view 
- Check that the correct Project is pre-selected in the search form

- Query for project/user/etc with many tasks
- Be amazed by the loading spinner

### Issues:
- fixes https://discuss.webknossos.org/t/option-to-open-the-list-of-tasks-for-a-project-is-missing/558/1

------
